src/components/Projects/ActionableLinksView.tsx

```diff
@@ -173,7 +173,7 @@ function ActionableLinksView({ links, projectId, proposalId, canEdit }: Props) {
           <ProjectSheetLinkItem
             label={t('project.sheet.general_info.links.proposal_link')}
             icon={<BlueLinkIcon />}
-            href={proposalUrl(proposalId)}
+            href={locations.proposal(proposalId)}
```

Comment:
Use proposalUrl instead as if user clicks it in the proposal, it doesn't work :(